pkg/registry/file.go

```diff
@@ -103,6 +105,32 @@ func translateCacheHandler(input, userID string) string {
 	return input
 }
 
+func translateOktetoRegistry(input string, ctx context.Context) string {
```

Comment:
I think for this case, we can just replace strings.
something like `okteto.dev` -> `registryurl/namespace` and `okteto.global` -> `registryurl/okteto` is a single

Reply:
Also, could you add unit tests for this function?